crowbar_framework/app/models/backup.rb

```diff
@@ -159,14 +159,19 @@ def save_archive
       return false
     end
 
+    if data.join("meta.yml").exist?
+      meta = YAML.load_file(data.join("meta.yml"))
```

Comment:
> What if it's not a valid yaml file?

Why wouldn't it be valid yaml when it is created by Crowbar? Sure I can add a check, but then we could also check each file in the backup. to some point we have to rely on the user to just take a backup that was created by crowbar, and not put together by hand...but I will add a check

@tboerger testing some invalid yaml i ran into a `Psych::SyntaxError`